src/schema/v2/notifications/__tests__/index.test.ts

```diff
@@ -12,7 +12,11 @@ describe("notificationsConnection", () => {
           status: "unread",
           date: "2022-08-22T21:15:49.000Z",
           object_ids: ["63036fafbe5cfc000cf358e3", "630392514f13a5000b55ecec"],
-          object: { artist: { id: "damien-hirst" } },
+          object: {
+            artist: { id: "damien-hirst", _id: "4d8b926a4eb68a1b2c0000ae" },
```

Comment:
I'm honestly not sure how these tests were passing before 🤔 this was failing both locally and CI because it thought we should always have an ID for the artist